kdwsdl2cpp/src/main.cpp

```diff
@@ -262,6 +286,16 @@ int main(int argc, char **argv)
     Settings::self()->setHelpOnMissing(helpOnMissing);
 
     KWSDL::Compiler compiler;
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
+    if (!pkcs12File.isEmpty()) {
+        if (!FileProvider::loadCertificate(pkcs12File, pkcs12Password)) {
+            fprintf(stderr, "Unable to load the %s certificate file\n", pkcs12File.toLocal8Bit().constData());
+            if (!pkcs12Password.isEmpty())
+                fprintf(stderr, "Please make sure that you have passed the correct password");
```

Comment:
I think this message is also applicable with a empty password, as an empty password could be the incorrect password.